docs/guides/request_storage.mdx

```diff
@@ -0,0 +1,136 @@
+        <CodeBlock className="language-python">
+            {RqWithCrawlerExplicitExample}
+        </CodeBlock>
+    </TabItem>
+</Tabs>
+
+## Request list
+
+The <ApiLink to="class/RequestList">`RequestList`</ApiLink> is a simpler, lightweight storage option, used when all URLs to be crawled are known upfront. It represents the list of URLs to crawl that is stored in a crawler run memory (or optionally in default <ApiLink to="class/KeyValueStore">`KeyValueStore`</ApiLink> associated with the run, if specified). The list is used for the crawling of a large number of URLs, when we know all the URLs which should be visited by the crawler and no URLs would be added during the run. The URLs can be provided either in code or parsed from a text file hosted on the web. The <ApiLink to="class/RequestList">`RequestList`</ApiLink> is typically created exclusively for a single crawler run, and its usage must be explicitly specified.
+
+:::warning
+
+The <ApiLink to="class/RequestList">`RequestList`</ApiLink> class is in its early version and is not fully
+implemented. It is currently intended mainly for testing purposes and small-scale projects. The current
+implementation is only in-memory storage and is very limited. It will be (re)implemented in the future.
+For more details, see the GitHub issue [crawlee-python#99](https://github.com/apify/crawlee-python/issues/99).
+For production usage we recommend to use the <ApiLink to="class/RequestQueue">`RequestQueue`</ApiLink>.
+
+:::
+
+The following code demonstrates the usage of the <ApiLink to="class/RequestList">`RequestList`</ApiLink>:
+
+<Tabs groupId="request_list">
+    <TabItem value="request_list_basic_example" label="Basic usage" default>
+        <CodeBlock className="language-python">
+            {RlBasicExample}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="request_list_with_crawler" label="Usage with Crawler">
+        <CodeBlock className="language-python">
+            {RlWithCrawlerExample}
+        </CodeBlock>
+    </TabItem>
+</Tabs>
+
+{/*
+
+## Which one to choose?
+
+TODO: write this section, once https://github.com/apify/crawlee-python/issues/99 is resolved
+
+*/}
+
+## Request-related helpers
+
+We offer several helper functions to simplify interactions with request storages:
+
+- The <ApiLink to="class/AddRequestsFunction">`add_requests`</ApiLink> function allows you to manually add specific URLs to the configured request storage. In this case, you must explicitly provide the URLs you want to be added to the request storage.
+- The <ApiLink to="class/EnqueueLinksFunction">`enqueue_links`</ApiLink> function is designed to find new URLs in the current page and add them to the request storage. You can specify different enqueue strategies or add excluding / including filters to limit the URLs that will be added to the request storage. See [Crawl website with relative links](../examples/crawl-website-with-relative-links) for more details.
```

Comment:
Maybe we could mention that it works fine without arguments... and we should probably mention selectors.

Reply:
There is already an example that demonstrates it works without arguments. Also, in my opinion, we should not describe the function interface & arguments in the guides (too much). For such cases should serve the API docs. It would be redundant and also hard to maintain.

Reply:
I mostly agree, it just feels weird to mention strategies and omit selectors...

Reply:
Updated